packages/perseus/src/components/tex-buttons.tsx

```diff
@@ -26,13 +26,13 @@ const basic: ReadonlyArray<ButtonSet> = [
         <span key="plus" style={slightlyBig}>
             +
         </span>,
-        "+",
+        "PLUS",
```

Comment:
Nice. So we're moving from the Mathquill command that should be sent to the conceptual `Key` and then the translator will figure out what that means for Mathquill later?  I like!